config/routes.rb

```diff
@@ -1,22 +1,14 @@
 Webapp::Application.routes.draw do
 
-  resources :home do
```

Comment:
Ah ok the way volunteers_controller uses `home` and `def home` is not clear, but it seems we don't use this. 👍